lib/base_service.js

```diff
@@ -183,6 +183,22 @@ BaseService.prototype._sanitizeConfig = function(conf, options) {
 };
 
 
+BaseService.prototype._replaceEnvVars = function(config) {
+    if(Buffer.isBuffer(config)) {
+        config = config.toString();
+    }
+    return config.replace(/\{\s*env\(([^,\s\)]+),?\s*([^\)]+)?\)\s*}/g, function(match, envName, defValue) {
```

Comment:
Which syntax are you using here? In bash the syntax is `{name:-default}` as far as I know. It that comma syntax coming from some standard?